includes/Model/Story.php

```diff
@@ -83,6 +83,19 @@ class Story {
 	 */
 	protected $publisher_logo;
 
+	/**
+	 * Poster source set sizes.
+	 *
+	 * @var string
+	 */
+	protected $poster_sizes = '';
+	/**
+	 * Poster source set.
+	 *
+	 * @var string
+	 */
+	protected $poster_srcset = '';
```

Comment:
Wait I meant to only remove only 1 newline of the double newlines you had here everywhere, but now all newlines were removed 🤔

Reply:
I just applied all the suggested changes.

Reply:
Fixed.